src/com/owncloud/android/authentication/AuthenticatorActivity.java

```diff
@@ -1605,13 +1607,18 @@ private void showServerStatus() {
             mServerStatusView.setVisibility(View.INVISIBLE);
 
         } else {
-            mServerStatusView.setText(mServerStatusText);
+            setStatusUrl();
             mServerStatusView.setCompoundDrawablesWithIntrinsicBounds(mServerStatusIcon, 0, 0, 0);
             mServerStatusView.setVisibility(View.VISIBLE);
         }
 
     }
 
+    private void setStatusUrl(){
+        String text = getString(R.string.auth_unknown_error_title) + "<br/>" + mHostUrlInput.getText().toString().trim() + "/status.php";

```

Comment:
I think that you should use formatted string:
```java
String text = getString(R.string.auth_unknown_error_title, mHostUrlInput.getText().toString().trim() + "/status.php");
```
and change auth_unknown_error_title string accordingly